# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: netwerk/test/unit/test_proxyconnect.js
# Commit: 23b864e14db0
# Full Hash: 23b864e14db0cf874d0e126b6299c8c44922dde3
# Author: Dana Keeler <dkeeler@mozilla.com>
# Date: 2022-11-16 09:31:06
# Regressor Bug: 1791633
# File Overlap Count: 1
# Description:
#   Bug 1791633 - separate nsITLSSocketControl from nsITransportSecurityInfo r=necko-reviewers,kershaw,jschanck
#   
#   Depends on D160311
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D160313
# ==============================================================================

diff -r 0bcba3375ec0 -r 23b864e14db0 netwerk/test/unit/test_proxyconnect.js
--- a/netwerk/test/unit/test_proxyconnect.js	Tue Nov 15 20:03:29 2022 +0000
+++ b/netwerk/test/unit/test_proxyconnect.js	Tue Nov 15 20:03:29 2022 +0000
@@ -255,7 +255,9 @@
     streamIn = transport
       .openInputStream(0, SEGMENT_SIZE, SEGMENT_COUNT)
       .QueryInterface(Ci.nsIAsyncInputStream);
-    streamOut = transport.openOutputStream(0, 0, 0);
+    streamOut = transport
+      .openOutputStream(0, 0, 0)
+      .QueryInterface(Ci.nsIAsyncOutputStream);
 
     streamIn.asyncWait(connectHandler, 0, 0, threadManager.mainThread);
   } catch (e) {